.gitattributes

```
@@ -1,2 +1,2 @@
 # Auto detect text files and perform LF normalization
-* text=auto
+* text eol=lf
```

Shouldn't it be:
```suggestion
* text=auto eol=lf
```
?

---

Oops. Good catch.